cypress_test/integration_tests/multiuser/calc/invalidations_spec.js

```diff
@@ -0,0 +1,52 @@
+/* global describe it cy beforeEach expect require afterEach */
+
+var helper = require('../../common/helper');
+var desktopHelper = require('../../common/desktop_helper');
+
+describe(['tagmultiuser'], 'Joining a document should not trigger an invalidation', function() {
+	var origTestFileName = 'invalidations.ods';
+	var testFileName;
+
+	beforeEach(function() {
+		testFileName = helper.beforeAll(origTestFileName, 'calc', undefined, true);
+		desktopHelper.switchUIToNotebookbar();
+	});
+
+	afterEach(function() {
+		helper.afterAll(testFileName, this.currentTest.state);
+	});
+
+	it('Join document', function() {
+		cy.cSetActiveFrame('#iframe1');
+
+		cy.cGet('#InsertMode').should('have.text', '\u00a0Insert mode: inactive\u00a0');
+		helper.typeIntoDocument('X');
+		cy.cGet('#InsertMode').should('have.text', 'Insert');
+		helper.typeIntoDocument('{enter}');
+		cy.cGet('#InsertMode').should('have.text', '\u00a0Insert mode: inactive\u00a0');
+		cy.cGet('input#addressInput').should('have.prop', 'value', 'A2');
+		helper.typeIntoDocument('{uparrow}');
+		// wait until round trip of cell address
+		cy.cGet('input#addressInput').should('have.prop', 'value', 'A1');
+
+		cy.cGet('.empty-deltas').then(($before) => {
```

Comment:
Why the dollar sign here, is that some kind of convention? It seems $ has no special meeting at a JS language level.

Reply:
This one is $before because the writer one I wrote in the past that I based this one off used $before, and that one was $before because I followed https://docs.cypress.io/guides/core-concepts/variables-and-aliases which used $btn as an example :-)

I think it is just convention and has no actual meaning.

Reply:
I see, thanks for the cypress reference.